src/test/java/jenkins/plugins/openstack/PluginTestRule.java

```diff
@@ -12,21 +15,18 @@
 import hudson.model.LoadStatistics;
 import hudson.model.Node;
 import hudson.model.TaskListener;
-import hudson.model.User;
```

Comment:
My IDE removed this unused import when I told it to tidy things up - it can be put back if you want to keep changes to a minimum.
(Same applies to the other imports that were removed in this change)